packages/backend/src/server/api/endpoints/i/webhooks/update.ts

```diff
@@ -66,7 +66,7 @@ export default class extends Endpoint<typeof meta, typeof paramDef> { // eslint-
 			await this.webhooksRepository.update(webhook.id, {
 				name: ps.name,
 				url: ps.url,
-				secret: ps.secret,
+				secret: ps.secret === null ? '' : ps.secret ? ps.secret : undefined,
```

Comment:
これだと `secret` に空文字列が入ったときに更新がかからなくなってしまうので、純粋にこう書いてしまって良さそうです。
```suggestion
				secret: ps.secret === null ? '' : ps.secret,
```

Reply:
仰る通りですね...修正(commit suggestion)させていただきました。